src/Data/List/Relation/Ternary/Appending.agda

```diff
@@ -42,7 +38,7 @@ module _ {A : Set a} {B : Set b}
 ------------------------------------------------------------------------
 -- Functions manipulating Appending
 
-_++_ : ∀ {cs₁ cs₂} → Pointwise L as cs₁ → Pointwise R bs cs₂ →
+_++_ : ∀ {cs₁ cs₂ : List C} → Pointwise L as cs₁ → Pointwise R bs cs₂ →
```

Comment:
There was a subtle bug in here due to the `variable` block! :-o

Reply:
Wow - super subtle.

Reply:
My bad: there was no bug. The bug was introduced by moving the sets from the `variable` block
to the telescope of parameters.